src/main/twirl/uk/gov/hmrc/hmrcfrontend/views/helpers/HmrcTimeoutDialogHelper.scala.html

```diff
@@ -30,7 +30,8 @@
   message: Option[String] = None,
   messageSuffix: Option[String] = None,
   keepAliveButtonText: Option[String] = None,
-  signOutButtonText: Option[String] = None
+  signOutButtonText: Option[String] = None,
+  synchroniseTabs: Option[Boolean] = None
```

Comment:
is this possible?

```suggestion
  synchroniseTabs: Boolean = timeoutDialogConfig.enableSynchroniseTabs
```

Reply:
I guess it probably isn't, just spotted that other optional stuff is using option wrapped values!

Reply:
@oscarduignan Yes, I was thinking that `Option[Boolean]` captures that this doesn't have to be set by the user, i.e. is optional. Having said, I have very mixed feelings about `Option[Boolean]` under any circumstances, so more than happy to revisit this.

Reply:
I'm easy either way, the more I squint the more it seems like not much of a muchness